azext_iot/tests/test_iot_ext_int.py

```diff
@@ -38,8 +40,27 @@
 
 
 class TestIoTHub(LiveScenarioTest):
+    def __init__(self, _):
+        from iotext_test_tools import DummyCliOutputProducer
+        super(TestIoTHub, self).__init__(_)
+        self.cli_ctx = DummyCliOutputProducer()
+
     def setUp(self):
         self._entity_names = None
+        for cg in LIVE_CONSUMER_GROUPS:
+            self.cmd('az iot hub consumer-group create --hub-name {} --resource-group {} --name {}'.format(LIVE_HUB, LIVE_RG, cg), checks=[self.check('name', cg)])
```

Comment:
The setUp at this level runs per test method. Should be moved to the test needing custom cg's.

Reply:
Cool. I thought that only ran once per test run.  I'll update